web_tool_script_3.R

```diff
@@ -221,6 +237,12 @@ create_interactive_report(
   header_dictionary = header_dictionary
 )
 
+has_arg <- utils::hasName(formals(create_executive_summary), "sector_order")
+if (has_arg) {
+  arg <- list(sector_order = sector_order())
+  arguments <- append(arguments, arg)
+}
+purrr::invoke(create_interactive_report, arguments)
```

Comment:
This will now work even if `create_interactive_report()` has no formal argument `sector_order`. 

(This is an intermediate refactoring step to break the dependency on create_interactive_report#313.)